nixos/modules/services/networking/gns3.nix

```diff
@@ -0,0 +1,261 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.gns3;
+
+  configFile = pkgs.writeText "gns3-server.conf" ''
```

Comment:
Is it a good idea to restrain arbitrarily the expressiveness of the configuration instead of following, e.g. RFC42, to offer the user to override any of the configuration here?

Reply:
@RaitoBezarius Hello. Thanks for the review!

Good point, I didn't know about RFC42.
I fixed this part of the code to use settings configuration block instead.